docs/guide/3-install-free5gc.md

```diff
@@ -172,6 +172,7 @@ sudo systemctl disable ufw # prevents the firewall to wake up after a OS reboot
 
         ```
         cd ~/free5gc/webconsole/frontend
+        corepack enable
```

Comment:
I think this should be enabled right after the yarn is installed.

Reply:
Which is after the ```sudo apt install -y nodejs``` since corepack is installed by default.

Reply:
Thanks for the suggestion. It's done in https://github.com/free5gc/free5gc.github.io/pull/103/commits/d63a1e99cf59c28568078e51b99dd606df54bb65